src/damavand/cloud/aws/resources/glue_component.py

```diff
@@ -118,28 +272,33 @@ def iceberg_database(self) -> aws.glue.CatalogDatabase:
             location_uri=f"s3://{self.iceberg_bucket.bucket}/",
         )
 
+    # Orchestration
+    def __create_glue_trigger(self, job: GlueJobDefinition) -> Optional[aws.glue.Trigger]:
+        """Return a Glue Trigger object."""
+        return aws.glue.Trigger(
+            f"{job.name}-glue-trigger",
+            type='SCHEDULED',
+            schedule=job.schedule,
+            actions=[aws.glue.TriggerActionArgs(
+                job_name=self.__get_job_name(job),
+            )],
+            start_on_creation=True
+        ) if job.schedule else None
```

Comment:
@callarelli93 This is a private method. I suggest to change it to a protected method. This hides the method on the object instances but when the user wants to inherit it to customize the class would be able to also customize this as well.

```suggestion
    def _create_glue_trigger(self, job: GlueJobDefinition) -> Optional[aws.glue.Trigger]:
        """Return a Glue Trigger object."""
        return aws.glue.Trigger(
            f"{job.name}-glue-trigger",
            type='SCHEDULED',
            schedule=job.schedule,
            actions=[aws.glue.TriggerActionArgs(
                job_name=self.__get_job_name(job),
            )],
            start_on_creation=True
        ) if job.schedule else None
```

Reply:
Clear will do!

Just learned about this convention, up until now I thought _ vs __ was a personal choice 🥲